dg/media/app_dashboards/js/loop_dashboard.js

```diff
@@ -2821,36 +2869,74 @@ function aggregator_payment_sheet(data_json, aggregator) {
             for (var j = 0; j < data_set.length; j++) {
                 if (data_set[j].indexOf(payments_gaddidar_contribution[i]['date']) != -1 && data_set[j].indexOf(payments_gaddidar_contribution[i]['mandi__name']) != -1) {
                     data_set[j][8] += parseFloat(payments_gaddidar_contribution[i]['amount']);
-                    data_set[j][9] = (data_set[j][9] - parseFloat(payments_gaddidar_contribution[i]['amount'])).toFixed(2);
+                    data_set[j][9] = parseFloat((data_set[j][9] - parseFloat(payments_gaddidar_contribution[i]['amount'])).toFixed(2));
                     break;
                 }
             }
         }
     }
 
+
+    
+
+    var formatVal = function (yourNumber) {
```

Comment:
this function is not required, javascript provide inbuilt functions to achieve the same. Please explore and do the needful